### PR TITLE
chore(connect): update release docs

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: docs/releases/connect-release.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ## @trezor/connect trezor.io release
-
 -   [x] From develop branch create a pull requests into branch `release/connect-v9`
+
+### After merging this PR
+-   [release/connect-v9 pipeline](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines?page=1&scope=branches&ref=release%2Fconnect-v9)
 -   [ ] Tested and approved by @trezor/qa. Typically using [this build](https://suite.corp.sldev.cz/connect/release/connect-v9/)
```
Comment (Hannsek) [inline docs/releases/connect-release.md]:
```suggestion
-   [ ] Tested and approved by @trezor/qa. Typically using [this build](https://suite.corp.sldev.cz/connect/release/connect-v9/) Changelog [here](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/CHANGELOG.md).
```

File: docs/releases/connect-prerelease.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## @trezor/connect prerelease
 
--   [x] Bump version in all @trezor/connect\* packages (except plugin packages). `yarn workspace @trezor/connect version:<beta|patch|minor|major>`
--   [ ] Make sure you have released all npm dependencies. If unsure run `node ./ci/scripts/check-npm-dependencies.js connect`. Please note that this script will report unreleased dependencies even for changes that do not affect runtime (READMEs etc.) If there are any unreleased npm dependencies you should [release them](./npm-packages.md)
--   [ ] Make sure [CHANGELOG](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/CHANGELOG.md) file has been updated
+-   [x] @mroz22 Bump version in all @trezor/connect\* packages (except plugin packages). `yarn workspace @trezor/connect version:<beta|patch|minor|major>`
```
Comment (Hannsek) [inline docs/releases/connect-prerelease.md]: I would maybe put the names behind the text, otherwise fine.

**Review by Hannsek — APPROVED** [review]

File: docs/releases/connect-release.md (hunk 1 above)
Comment (mroz22) [reply docs/releases/connect-release.md]: link should be https://github.com/trezor/trezor-suite/blob/release/connect-v9/packages/connect/CHANGELOG.md

Comment (Hannsek) [reply docs/releases/connect-release.md]: ahhh, right

**Pull request merged**